docs/blog/_posts/2019-03-01-13th-dotty-milestone-release.md

```diff
@@ -0,0 +1,407 @@
+---
+layout: blog-page
+title: Announcing Dotty 0.13.0-RC1
+author: Aggelos Biboudis
+authorImg: /images/aggelos.jpg
+date: 2019-03-01
+---
+
+Hello hello! This is the second release for 2019, let's call it the _Contextual_
+release and you will understand why we are super excited in a bit! ✨🎊🎉
+
+Without further ado, today we release the version 0.13.0-RC1 of the Dotty compiler.
+This release serves as a technology preview that demonstrates new language features and the
+compiler supporting them.
+
+Dotty is the project name for technologies that are considered for inclusion in Scala 3. Scala has

```

Comment:
are being considered

Reply:
that are considered worthy of inclusion

Reply:
I think one purpose of this sentence is to reassure people that not everything that has been merged in Dotty is a done deal, it still needs to be considered by the community and go through the SIP committee and so forth. so "are considered" gives the wrong impression, I think.